### DISCUSSION TITLE
Uncomplete GitHub profile?

### DISCUSSION THREAD

**Discussion by MateoDaruich** [community/community#57377] [GitHub Education]
### Hi everyone,

I keep getting denied from the developer pack, thankfully, the list of reasons seems to be growing smaller with every request but I'm stuck at the following:

"You are unlikely to be verified until you have completed your [GitHub user profile](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/customizing-your-profile/personalizing-your-profile) with your full name as it appears in your academic affiliation document plus a short bio."

I really don't understand what more I could add to my profile to prove it, any help would be appreciated. 
Thanks in advance.

**Comment by Hajubix** [comment]
Im stuck with the same problem. Sorry I cant help

**Comment by TithiB-del** [comment]
Hi! just a small suggestion-
make sure that the full name is entered correctly in the first name & last name fields.
Also, the name needs to be exactly same as it is on your institution approved id card to verify that you are a student.

**Reply by Hajubix** [reply]
I already have my full name, the one on my school certificate, in my GitHub. I also have completed my account info as far as I know is possible. The document I tried scanning was a school certificate which had my name and the name of the school on it.

**Comment by chriskthomas** [comment]
They mean for you to put your name here:
![image](https://github.com/community/community/assets/135461249/b4c844ba-b96c-46eb-9bed-600cc5a69728)

Your username can be whatever you want.

**Reply by Hajubix** [reply]
I had a hyphen in between. Changed it now

**Comment by jupmorenor** [comment]
Maybe having some active repos would help for both of you.

**Comment by samus-aran** [comment]
Hi @MateoDaruich 👋 

Often this common pre-processing warning is displayed because the application detects differences between your profile information and the evidence you’ve supplied. We do request that students can demonstrate that their GitHub account corresponds with the evidence given. The verification process looks at your GitHub username, email, and name (as well as other points) to cross-reference the evidence submitted, so be sure that they match to ensure a smoother process.

We would also suggest that when you update your profile information that you log out and log back into GitHub before re-applying. 👈 I suspect this section will be helpful.
